packages/core/src/index.ts

```diff
@@ -6,3 +6,18 @@
 
 // we export any non component code such as utilities at the root '@clr/core'
 export { CommonStringsService, ClrCommonStrings, commonStringsDefault, runCssVarsPolyfill } from '@clr/core/internal';
+
+// type global attribute utilities
+declare global {
+  interface HTMLElement {
+    'cds-text': string;
+    'cds-layout': string;
+    'cds-list': string;
+  }
+
+  interface HTMLElementTagNameMap {
+    'cds-card': HTMLElement;
+    'cds-demo': HTMLElement;
+    'cds-placeholder': HTMLElement;
+  }
```

Comment:
The type checker for the templates would complain about these since they are not defined elements. I'm still looking around see if there is a better spot these could exist. Maybe a types.d.ts in the directory would work?